.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    branches:
+      - release/v*
+jobs:
+  aws-release:
+    name: Release zip file for lambda deployment
+    runs-on: ubuntu-latest
+    needs: bump-version
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```

Comment:
You can also store `ZIP_ASSET_NAME` instead (which contains the version), because that's the only place it's being used in the rest of the job.

Reply:
Used it to make code cleaner. Thanks for the tip.